caravel/utils.py

```diff
@@ -486,3 +486,15 @@ def __exit__(self, type, value, traceback):
         except ValueError as e:
             logging.warning("timeout can't be used in the current context")
             logging.exception(e)
+
+def wrap_clause(sql):
```

Comment:
the doctest fails because you return a `sqlalchemy.text` object. 

I'd make this function more generic and call it `wrap_in_parentheses(s)` and wrap clauses systematically (regardless of the OR in there). Note that the `'OR' in ` currently would trigger on something like 'my_field_has_the_letter_or_in_it > 0', but that's fine, we should just always put parentheses around the user clause when provided